using_images/s2i_images/php.adoc

```diff
@@ -120,8 +120,13 @@ The following environment variable sets its equivalent property value in the opc
 |Variable name |Description |Default
 
 |`*OPCACHE_MEMORY_CONSUMPTION*`
-|The OPcache shared memory storage size.
+|The link:http://php.net/manual/en/book.opcache.php[OPcache] shared memory storage size.
```

Comment:
My reasoning is that when a user reads the docs, he might not understand what OPcache is, or what does limiting its shared memory storage entail, that's all. But ok, I guess linking to OPcache is enough.